content/en/docs/tasks/job/indexed-parallel-processing-static.md

```diff
@@ -66,13 +66,21 @@ program accepts a file as an argument and prints its content reversed.
 rev data.txt
 ```
 
-For this example, you'll use the `rev` tool from the
+You'll use the `rev` tool from the
 [`busybox`](https://hub.docker.com/_/busybox) container image.
 
+As this is only an example, each Pod only does a tiny piece of work (reversing a short
+string). In a real workload you might, for example, create a Job that represents
+ the
+task of producing 60 seconds of video based on scene data.
+Each work item in the video rendering Job would be to render a particular
+frame of that video clip. Indexed completion would mean that each Pod in
+the Job knows which frame to render and publish, by counting frames from
+the start of the clip.
+
 ## Define an Indexed Job
 
-Here is a job definition. You'll need to edit the container image to match your
-preferred registry.
+Here is a sample Job manifest that uses `Indexed` completion mode:
```

Comment:
nit: What is "Indexed" completion mode? Should this be capitalized and use code formatting?

Reply:
The page prerequisites direct the reader to read up about Job first, which (in the v1.21 docs) explains that mode. We could link directly to that part of the page to emphasize this?

Reply:
I don't think it's necessary. The following paragraph is the example, which shows how the completion mode is set in the spec.